source/catalog.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+  </head>
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo" href="index.html" title="Перейти на главную страницу">
+        <img src="img/logo.svg" alt="Логотип Седоны." width="185" height="96" >
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list nav-list">
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="#">
+                Главная
+              </a>
+            </li>
+            <li class="nav-list__item--current">
+              <a class="nav-list__link" href="#">
+                Фото и видео
+              </a>
+            </li>
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="#">
+                Форма отзыва
+              </a>
+            </li>
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="https://htmlacademy.ru/">
+                HTML Academy
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="catalog-container">
+      <h1 class="visually-hidden">Седона</h1>
+      <div class="hero-block-container">
+        <img class="hero-block-container__logo" src="#"  alt="Логотип-приветствие.">
+      </div>
+      <section class="header--gallary">
```

Comment:
Модификаторы используются всегда вместе с блоком
```suggestion
      <section class="header header--gellary">
```